### ISSUE TITLE
CTRL+Z empties the content

### ISSUE THREAD

**Issue by supunkavinda** [uiwjs/react-codemirror#261]
If you focus on the editor and press `CTRL+Z` the content dissapears. I think this is not expected behavior. This issue happens in both [v3](https://raw.githack.com/uiwjs/react-codemirror/doc3/index.html) and [v4](https://uiwjs.github.io/react-codemirror/) demos in `react-codemirror`. 

This doesn't happen in the [codemirror demo](https://codemirror.net/), so I think it's an issue with `react-codemirror` package.

Any fixes or workarounds?

**Comment by jaywcjlove** [comment]
@supunkavinda I didn't reproduce your problem.

**Comment by supunkavinda** [comment]
* Go to the demo: https://uiwjs.github.io/react-codemirror/
* Focus somewhere in the code editor
* Click `Ctrl + Z`. It empties the code editor

It is happening for me on Safari/Chrome/Firefox on macOS

**Comment by jaywcjlove** [comment]
This is codemirror's [history](https://codemirror.net/6/docs/ref/#history) feature. The default display interface, the editor shows the example, and the `value` is changed once.

**Comment by supunkavinda** [comment]
Yeah, but it doesn't happen on CodeMirror demos (trying ctrl+z on these demo code editors, they don't do anything):

https://codemirror.net/
https://codemirror.net/6/

General, why should `ctrl + z` should empty content? 🤔 

I think in react-codemirror, CodeMirror is getting the empty value as the first value.

Does that make sense?

**Comment by jaywcjlove** [comment]
@supunkavinda There is no such problem in the example.

https://codesandbox.io/embed/react-codemirror-example-codemirror-6-slvju?fontsize=14&hidenavigation=1&theme=dark
